src/Swarm/Game/CESK.hs

```diff
@@ -263,7 +263,7 @@ data CESK
     Up Exn Store Cont
   | -- | The machine is waiting for the game to reach a certain time
     --   to resume its execution.
-    Waiting Integer CESK
+    Waiting Number CESK
```

Comment:
Oh, wait, I think I get it --- `Number` is the Haskell type that is used internally to represent Swarm values of type `int`.  So since `wait : int -> cmd unit`, therefore `Waiting` has to take a `Number` because `wait infinity` is now valid Swarm code.  Did I understand it right?

Reply:
Yes, it might be less confusing for us to rename `int` to `number`.

Besides, it will eventually be useful to wait indefinitely and get woken up by some other robot.